db/migrate/20150224140913_create_community_translations.rb

```diff
@@ -0,0 +1,12 @@
+class CreateCommunityTranslations < ActiveRecord::Migration
+  def change
+    create_table :community_translations do |t|
+      t.integer :community_id,    null: false
+      t.string  :locale,          null: false
```

Comment:
Yeah, you are right. 
(It seems that we have the same 256 in other translation-tables too, but hopefully rows are not using it all :)